src/bin/bootstrap/CommandBootstrap.ts

```diff
@@ -36,11 +38,28 @@ class CommandBootstrap extends CommandPolykey {
           }
         }
       }
-      await bootstrapPolykeyState(options.nodePath, password);
+      const recoveryCode = await binUtils.getRecoveryCode(
+        options.recoveryFile,
+        this.fs,
+      );
+      const newRecoveryCode = await bootstrapUtils.bootstrapState({
+        nodePath: options.nodePath,
+        password,
+        recoveryCode,
+        fresh: options.fresh,
+        fs: this.fs,
+        logger: this.logger,
+      });
+
       process.stdout.write(
         binUtils.outputFormatter({
           type: options.format === 'json' ? 'json' : 'list',
-          data: [`Polykey bootstrapped at Node Path: ${options.nodePath}`],
+          data: [
+            `Polykey bootstrapped at Node Path: ${options.nodePath}`,
```

Comment:
Same issue here this should be `this.logger.info`.

Reply:
I've replicated what has been done in `agent/CommandStart.ts`